roles/openshift_logging_curator/files/actions.yaml

```diff
@@ -0,0 +1,37 @@
+# ---
+# Remember, leave a key empty if there is no value.  None will be a string,
+# not a Python "NoneType"
+#
+# Also remember that all examples have 'disable_action' set to True.  If you
+# want to use this action as a template, be sure to set this to False after
+# copying it.
+# actions:
+#   1:
+#     action: delete_indices
+#     description: >-
+#       Delete indices older than 30 days (based on index name), for .defaults
+#       and .operations indices. Ignore the error if the filter does not result in an
+#       actionable list of indices (ignore_empty_list).
+#     options:
+#       # Swallow curator.exception.NoIndices exception
+#       ignore_empty_list: True
+#       # In seconds, default is 30
```

Comment:
This is the timeout to receive a response of the curation action?  @portante should we increase the default?

Reply:
Timeout for the operation to finish. Can take hours in large clusters.
The comment is wrong, it's CURATOR_TIMEOUT=300 in the Dockerfile

I will also make sure this is configurable in ansible.

Reply:
@portante What is your experience? Does 10 minutes limit cover most of deployments or should I increase it?